src/web/components/BusinessContacts/BusinessContactsTable.scss

```diff
@@ -24,9 +24,12 @@
       border: none;
     }
 
-    td.action {
+    th.action {
+      text-align: center;
```

Comment:
Making the th center and the td (below) flex-end means it gets out of alignment as the browser gets to full-width.
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/111951647/b0654d91-d5e8-48d8-ad28-986b06cb3523)